docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,3414 @@
+
+[float]
+[[enhancement-v8.7.0]]
+=== Enhancements
+Alerting::
+* Bring flapping status and settings in o11y {kibana-pull}150483[#150483]
+* Redesign all cases list select modal {kibana-pull}149851[#149851]
+* RenderCustomActionsRow with named params instead of args {kibana-pull}149304[#149304]
+* Adds new column `Updated on ` in `all cases list ` table. This column can be sorted and can persist sorting options {kibana-pull}149116[#149116]
+* Users can now click a button on Case Detail and All Cases List to copy a case's UUID to the clipboard {kibana-pull}148962[#148962]
+* Sorting, severity filter and status filter are now persisted in the URL and local storage for the all cases list {kibana-pull}148549[#148549]
+* Sorting by status, severity, and title is now allowed in the all-cases list {kibana-pull}148193[#148193]
+* See "My recently assigned cases" to the recent cases widget of Security Solution dashboard {kibana-pull}147763[#147763]
+* Adds the ability to bulk edit assignees on multiple cases {kibana-pull}146907[#146907]
+* Save draft user comment {kibana-pull}146327[#146327]
+* Ram 145739 use bulk enable disable in UI {kibana-pull}145928[#145928]
+* Create generic retry if function {kibana-pull}145713[#145713]
+* Return rules from bulk enable {kibana-pull}145391[#145391]
+* Create bulk disable endpoint {kibana-pull}145179[#145179]
+* Adding group by options to ES query rule type {kibana-pull}144689[#144689]
+
+APM::
+* Allow alert table to show new alert status on apm {kibana-pull}150500[#150500]
+* Switch get environment function to use terms_enum api {kibana-pull}150175[#150175]
+* Use (rolled up) service metrics for service inventory {kibana-pull}149938[#149938]
+* Adds KQL filter bar to the service map page {kibana-pull}149900[#149900]
+* Integrate Alert Search bar in alert tab {kibana-pull}149610[#149610]
+* Added Azure Functions support in the APM UI {kibana-pull}149479[#149479]
+* Adds 404 page {kibana-pull}149471[#149471]
+* Single-click setup from Kibana {kibana-pull}148959[#148959]
+* Update the sparklines to support the bar chart graph style {kibana-pull}148702[#148702]
+* Flamegraph legend {kibana-pull}147910[#147910]
+* Adds api key to APM package policies {kibana-pull}147650[#147650]
+* Only renders waterfall items up until 3 levels {kibana-pull}147569[#147569]
+* Improve span links navigation {kibana-pull}147426[#147426]
+* Update default refresh interval to 60 seconds {kibana-pull}146791[#146791]
+* Adds pagination to source map API {kibana-pull}145959[#145959]
+* Ability to offset point labels on maps {kibana-pull}145773[#145773]
+
+Dashboard::
+* Unified panel options pane {kibana-pull}148301[#148301]
+* Anchor time slider to start {kibana-pull}148028[#148028]
+* Show document count beside options list suggestions {kibana-pull}146241[#146241]
+
+Discover::
+* Show "Copy value" button for any grid cell {kibana-pull}149525[#149525]
+* Integrate unified field list filters into Discover {kibana-pull}148547[#148547]
```

Comment:
```suggestion
* Align field list filters UI between Discover and Lens {kibana-pull}148547[#148547]
```
Explaining what this does might be more helpful here -- I'm not sure Unified Field List will mean much to anyone reading this.